### DISCUSSION TITLE
Model completely independent sources?

### DISCUSSION THREAD

**Discussion by kaliif** [optimuslib/optimus#35] [Q&A] [answered]
Is there a way to model completely independent sources? There's an option for a bowl array and a piston but what I'd like is something like multiple pistons, say on a line or other, not necessarily spherical configuration. Is this possible?

**Comment by pgelat** [comment, marked as answer]
Thanks for your query. This is a feature that we plan to implement very soon. In first instance, we want to generalise the bowl array of circular piston sources to a planar array. Would this address your query or would you require a more generalised configuration?

**Comment by kaliif** [comment]
Thanks for the prompt reply. A planar array would indeed solve the most immediate configurations I had in mind.

**Reply by pgelat** [reply]
Great, we will get to work and hopefully push a new version onto GitHub by the end of next week.

**Reply by pgelat** [reply]
We have now added the planar array option as well as a notebook demonstrating its functionality. Hope this helps.
